xarray/tests/test_backends.py

```diff
@@ -2243,60 +2243,53 @@ def open(self, path, **kwargs):
                           **kwargs) as ds:
             yield ds
 
+    def test_uamiv_format_read(self):
```

Comment:
Something in this test is triggering a warning when I run it:
```
xarray/tests/test_backends.py::PseudoNetCDFTest::test_uamiv_format_read
  /Users/shoyer/miniconda3/envs/xarray-py36/lib/python3.6/site-packages/PseudoNetCDF/conventions/ioapi/_ioapi.py:134: UserWarning: IOAPI_ISPH is assumed to be 6370000.; consistent with WRF
    warn('IOAPI_ISPH is assumed to be ' + ENV_IOAPI_ISPH + '; consistent with WRF')
```

Can you please silence this warning by using a context manager: https://docs.python.org/2/library/warnings.html#temporarily-suppressing-warnings

Reply:
Done.